objc/src/main/java/org/robovm/objc/ObjCClass.java

```diff
@@ -112,6 +112,10 @@ private ObjCClass(long handle, Class<? extends ObjCObject> type, String name, bo
         this.type = type;
         this.name = name;
         this.custom = custom;
+        
+        if (custom) {
+            ObjCObject.ObjectOwnershipHelper.registerClass(this);
```

Comment:
I think it would be better to have this call at the end of the `ObjCClass.register()` method. Just after the `ObjCClass` instance is created.